### PR TITLE
Feature/setup

### PR DESCRIPTION
**Pull request by elic4vet**
I changed the structure of the image container

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,10 +1,10 @@
 <!DOCTYPE html>
 <html lang="en">
   <head>
-    <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta charset="UTF-8" />
```
Comment (basiacarvalho) [inline index.html:4]: "meta" tag can't have the slash "/" at the end - take it out (make sure you don't use prettier)

File: index.html [hunk 2]
```diff
@@ -1,10 +1,10 @@
...
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
```
Comment (basiacarvalho) [inline index.html:5]: "meta" tag can't have the slash "/" at the end - take it out (make sure you don't use prettier)

**Comment by elic4vet** [comment]
I fixed the issue on the first section on the main page

File: index.html [hunk 3]
```diff
@@ -1,10 +1,10 @@
...
     <title>CodeStudio</title>
-    <link rel="stylesheet" href="./css/style.css">
+    <link rel="stylesheet" href="./css/style.css" />
```
Comment (basiacarvalho) [inline index.html:7]: "link" tag can't have the slash "/" at the end - take it out (make sure you don't use prettier)

File: index.html [hunk 4]
```diff
@@ -25,65 +27,71 @@
...
             <li class="offer__item">
-              <img 
-                class="offer__img img-placeholder" 
-                src="./images/html.jpg" 
+              <img
+                class="offer__img img-placeholder"
+                src="./images/html.jpg"
                 alt="laptop's keyboard as a background with a word - html, in the center"
-              >
+              />
```
Comment (basiacarvalho) [inline index.html:59]: "img" tag can't have the slash "/" at the end - take it out (make sure you don't use prettier)

File: index.html [hunk 5]
```diff
@@ -25,65 +27,71 @@
...
             <li class="offer__item">
-              <img 
-                class="offer__img" 
-                src="./images/css.jpg" 
+              <img
+                class="offer__img"
+                src="./images/css.jpg"
                 alt="a fragment of CSS code displayed on the computer's screen"
-              >
+              />
```
Comment (basiacarvalho) [inline index.html:70]: "img" tag can't have the slash "/" at the end - take it out (make sure you don't use prettier)

File: index.html [hunk 6]
```diff
@@ -25,65 +27,71 @@
...
             <li class="offer__item">
-              <img 
-                class="offer__img" 
-                src="./images/javascript.jpg" 
+              <img
+                class="offer__img"
+                src="./images/javascript.jpg"
                 alt="a fragment of JavaScript code displayed on the computer's screen"
-              >
+              />
```
Comment (basiacarvalho) [inline index.html:81]: "img" tag can't have the slash "/" at the end - take it out (make sure you don't use prettier)

File: index.html [hunk 7]
```diff
@@ -25,65 +27,71 @@
...
-              <p class="offer__advert">Give your website great functionality.</p>
+              <p class="offer__advert">
+                Give your website great functionality.
+              </p>
             </li>
             <li class="offer__item">
-              <img 
-                class="offer__img" 
-                src="./images/react.jpg" 
+              <img
```
Comment (basiacarvalho) [inline index.html:90]: "img" tag can't have the slash "/" at the end - take it out (make sure you don't use prettier)

**Review by basiacarvalho — CHANGES_REQUESTED** [review]
I left comments - tags have "/" slashes that have to be removed

File: pages/about.html [hunk 8]
```diff
@@ -4,7 +4,7 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>About us</title>
-    <link rel="stylesheet" href="../css/style.css">
+    <link rel="stylesheet" href="../css/style.css" />
```
Comment (basiacarvalho) [inline pages/about.html:7]: slash

File: pages/course.html [hunk 9]
```diff
@@ -1,10 +1,10 @@
 <!DOCTYPE html>
 <html lang="en">
   <head>
-    <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta charset="UTF-8" />
```
Comment (basiacarvalho) [inline pages/course.html:4]: slash

File: pages/course.html [hunk 10]
```diff
@@ -1,10 +1,10 @@
...
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
```
Comment (basiacarvalho) [inline pages/course.html:5]: slash

File: pages/course.html [hunk 11]
```diff
@@ -1,10 +1,10 @@
...
     <title>Our Course</title>
-    <link rel="stylesheet" href="../css/style.css">
+    <link rel="stylesheet" href="../css/style.css" />
```
Comment (basiacarvalho) [inline pages/course.html:7]: slash

File: pages/course.html [hunk 12]
```diff
@@ -26,70 +28,75 @@
...
-              class="course__img-description img-placeholder" 
-              src="../images/course-woman.jpg" 
+            <img
+              class="course__img-description img-placeholder"
+              src="../images/course-woman.jpg"
               alt="fragments of code on various screens displayed as an overlay 
               over a woman working on the computer by her office desk with some 
               personal items"
-            >
+            />
```
Comment (basiacarvalho) [inline pages/course.html:48]: slash

File: pages/course.html [hunk 13]
```diff
@@ -26,70 +28,75 @@
...
         <div class="container benefits-container">
-          <img 
-            class="benefits-img img-placeholder" 
+          <img
+            class="benefits-img img-placeholder"
             src="../images/course-man.jpg"
             alt="fragments of code displayed as an overlay over a man working 
             on the computer by his home desk with some personal items"
-          >
+          />
```
Comment (basiacarvalho) [inline pages/course.html:99]: slash

**Review by basiacarvalho — CHANGES_REQUESTED** [review]
A few more slashes

File: index.html (hunk 1 above)
Comment (basiacarvalho) [inline index.html:4]: slash

File: index.html [hunk 14]
```diff
@@ -25,30 +27,34 @@
...
+            <img
+              class="hero-img"
+              src="./images/heroimg.jpg"
               alt="an open laptop with code displayed on a desk with a plant and a mug"
-            >
+              >
+          </div>
         </div>
       </section>
+
```
Comment (basiacarvalho) [inline index.html:49]: empty line

**Review by basiacarvalho — CHANGES_REQUESTED** [review]

File: index.html [hunk 15]
```diff
@@ -4,7 +4,7 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>CodeStudio</title>
-    <link rel="stylesheet" href="./css/style.css">
+    <link rel="stylesheet" href="./css/style.css" />
```
Comment (basiacarvalho) [inline index.html:7]: one more slash

**Review by basiacarvalho — CHANGES_REQUESTED** [review]
Last slash to do :)

**Review by basiacarvalho — APPROVED** [review]

**Pull request merged**